src/components/KeyValProp/KeyValProp.tsx

```diff
@@ -0,0 +1,29 @@
+import { Typography, TypographyProps } from '@material-ui/core';
+import { FC } from 'react';
+import * as React from 'react';
+
+export type KeyValPropProps = TypographyProps & {
+  label?: string;
+  LabelProps?: TypographyProps;
+  value?: string | number | null;
+  ValueProps?: TypographyProps;
+};
+
+export const KeyValProp: FC<KeyValPropProps> = ({
```

Comment:
I left it in the one file un-exported so I didn't really care about the name. But then it got copied to another file, so it's good that you've extracted it out. The name is leaves something to be desired though...it's ambiguous.

I'm thinking about a more descriptive name...any thoughts?

Reply:
FieldDisplay? Maybe that sounds too much like it belongs in a form

Reply:
Yeah I agree. Field is associated with forms right now. We do seem to be using _display_ for this kind of thing though. `DisplayProperty`. Or maybe something more indicates this is a simple or terse UI component. What about `DisplaySimpleProperty`?